index.html

```diff
@@ -276,6 +276,45 @@ <h3 class="im-qaisar">Glad to see you!</h3>
           </ul>
         </div>
     </section>
+    <section>
+      <div class="contact-me-form-container">
+        <h1 class="contact-me-form-heading">Contact me</h1>
+        <p class="contact-me-form-paragraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/mdovrdrq" method="post" class="contact-me-form">
+          <input
+            class="contact-me-form-text-input"
+            required
+            maxlength="30"
+            name="name"
+            type="text"
+            placeholder="Your good name"
+          />
+          <input
+            class="contact-me-form-text-input"
+            required
+            name="email"
+            type="email"
+            placeholder="Your email address"
+          />
+          <textarea
+            class="contact-me-form-text-input"
+            required
+            maxlength="500"
+            name="message"
+            cols="30"
+            rows="9"
+            placeholder="Write your message here"
+          ></textarea>
+          <button class="contact-me-form-button" type="submit">
+            Get in touch
+          </button>
+        </form>
+      </div>
+    </section>
 
 <script>
     var coll = document.getElementsByClassName("collapsible");
```

Comment:
- [OPTIONAL] This is not among the requirement for this milestone, But I'll like to point out to you that, following the best practices, it is not advisable to have your javascript code inside your HTML file. I'll suggest you create a javascript file and move the code there. :+1: :100: